static/src/javascripts/projects/commercial/modules/dfp/load-advert.ts

```diff
@@ -4,28 +4,31 @@ import { prebid } from '../header-bidding/prebid/prebid';
 import { stripDfpAdPrefixFrom } from '../header-bidding/utils';
 import type { Advert } from './Advert';
 
-const forcedSlotSize = (advert: Advert, hbSlot: HeaderBiddingSlot) => {
-	// We only fiddle with top-above-nav hbSlot(s)
+// Force the refreshed advert to be the same size as the first
+const retainTopAboveNavSlotSize = (
+	advertSize: Advert['size'],
+	hbSlot: HeaderBiddingSlot,
+): HeaderBiddingSlot[] => {
 	if (hbSlot.key !== 'top-above-nav') {
 		return [hbSlot];
 	}
-	// For top-above-nav slots, we force the refreshed
-	// to be the same size as the first display
+
+	// No point forcing a size, as there is already only one possible (mobile/tablet).
+	// See prebid/slot-config.js
 	if (hbSlot.sizes.length === 1) {
-		// No point forcing a size, as there is already only one
-		// possible (mobile/tablet). See prebid/slot-config.js
 		return [hbSlot];
 	}
 
-	if (Array.isArray(advert.size)) {
-		return [
-			Object.assign({}, hbSlot, {
-				sizes: [[advert.size[0], advert.size[1]]],
-			}),
-		];
+	// If advert.size is not an array, there is no point having this hbSlot
+	if (!Array.isArray(advertSize)) {
+		return [];
 	}
-	// No point having this hbSlot, as advert.size is not an array
-	return [];
+
+	return [
+		Object.assign({}, hbSlot, {
+			sizes: [[advertSize[0], advertSize[1]]],
+		}),
```

Comment:
Think this could be converted to a spread:

```suggestion
		{
		  ...hbSlot
			sizes: [[advertSize[0], advertSize[1]]],
		},
```